{"name": "test_login_error[test_info0]", "status": "passed", "description": "登录失败", "parameters": [{"name": "test_info", "value": "{'mobile': '', 'password': '12', 'expected': '账号不能为空'}"}], "start": 1595077724521, "stop": 1595077730220, "uuid": "01cf2a92-e652-4900-bbf6-c8aea2c6c57c", "historyId": "1a42a92fe9250ffdeb64eef1609dd764", "testCaseId": "5fd5da3b3ca8fdf751feb802572e5b42", "fullName": "tests.test_login.TestLogin#test_login_error", "labels": [{"name": "tag", "value": "login"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-83PFO6C"}, {"name": "thread", "value": "12272-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}